### ISSUE TITLE
[TODO] Characters & Dynasties Work

### ISSUE THREAD

**Issue by RowanMaBoot** [Altemio/SevenKingdoms-master#6]
The_Seven_Kingdoms\common\dynasties -> This folder will contain dynasties. We'll want to create dynasties for our characters first. Dynasties need a unique ID!

The_Seven_Kingdoms\history\characters -> This folder will contain characters. Characters need a unique ID!

We will need to assign a name range for different Kingdoms, and put characters in separate files. We've got the test_dude/s.txt file for now, but down the line it'd be nice to have files like daynes.txt

For now the characters will be barebones. I'd like to add new traits down the line for characters, which we will have to add to any characters we've made.

**Comment by Vierwood** [comment]
Well first we need to fix the start date before we start adding in characters.

**Comment by RowanMaBoot** [comment]
I'll be sorting that out today, hopefully.

Might be too busy to do it though - however, you guys can start creating characters and dynasties, treating 6500.1.01 as the start date. They can be born before then, of course.

**Comment by tannerflick4** [comment]
I deleted Baelish's ancestor char for now, as I don't think it fits? I'll let you guys decide.

Also, any chars I add feel free to edit them.

**Comment by ZeroFighterR** [comment]
Indeed, I dont think it would fit in our timeline.

Working slowly on adding characters to the North

**Comment by tannerflick4** [comment]
Ok so Iron Islands are finished.

Are we essentially done with chars and dynasties? Just let me know what I should work on next...

**Comment by Vierwood** [comment]
Gonna start work on the Crownlands and Reach next week, I'll also add 15 or so more houses to the Vale.

**Comment by tannerflick4** [comment]
Sounds good. I plan on working on the Wall and then the wildlings.

**Comment by Vierwood** [comment]
**IMPORTANT:** Seeing as I've been basically the only one cramming for the release date. I could really use some help. If anyone could create and land all the Dornish counts that would make my day...

**Comment by tannerflick4** [comment]
Gotcha.

**Comment by tannerflick4** [comment]
Need a extended range for Dornish characters, as I am nearing 1200 which is where the Reach chars start.

**Comment by Vierwood** [comment]
Go ahead and create another .txt file for them.

**Comment by tannerflick4** [comment]
ok sounds good

**Comment by RowanMaBoot** [comment]
5700-5799 is free, or you can use 5700 - 5899 if you need more. Name the file 57-58 OR 59 depending on range. ;)

**Comment by tannerflick4** [comment]
Ok so the Stepstone pirates are close to being all set up for the main bookmark. Yay!

But, any requests for what I should do character-wise next? My initial thought was to head over to Essos and start working with Lys ... but I am open to suggestions.

**Comment by RowanMaBoot** [comment]
I think that would be good. Especially with the new patch coming, we'll be able to make the region real interesting with the china mechanics!

**Comment by tannerflick4** [comment]
great!

**Comment by Vierwood** [comment]
I'm going to overhaul Dorne and add in new GFX and Kingdoms:

Kingdom of the Boneway
Kingdom of the Redmarch
Kingdom of Skyreach
Kingdom of Blackmont
Principality of Lemonwood
Kingdom of Tor
Kingdom of Sandstone

Any ideas for more stuff?

**Comment by TitanRogue** [comment]
@Vierwood 
How do you make the coa?

**Comment by Vierwood** [comment]
@TitanRogue what do you mean? The code,  or the physical coa?

**Comment by TitanRogue** [comment]
The physical coa. Like wich effects to use for the "flag style".

**Comment by Vierwood** [comment]
Well I mainly copied them over from the Wiki of Ice and Fire before going in and cleaning them up. A few I made by myself, but their quality is questionable at best.

**Comment by TitanRogue** [comment]
@Altemio 
I actually don't know where to put it in and this issue seems to fit: The "house" trait is set to agnatic = yes so only males can inherit it. When having a daughter only the trait is lost. We could make a cognatic trait with less prestige, etc. but when having an matrilineal marriage you get the "full" trait by an event.

TheWindThief has made some notes viewable on Discord about lore and bugs.

**Comment by RowanMaBoot** [comment]
@TitanRogue 

I'd probably tweak to to work better, since it was basically just thrown in for the sake of it, if it weren't for the upcoming update:

[https://forumcontent.paradoxplaza.com/public/379340/Clipboard03.png

PDX seems to be adding a bloodline system of some sort themselves - so will be best to just use that system than actually turn the trait we have into a proper system (plus since it is dynamically named, we have some issues with the trait anyway).

**Comment by TitanRogue** [comment]
@Altemio
Yeah, the bloodtrait system pdx wants to add looks interesting but it is shown in an interface only. I suggest we make the trait agnatic and cognatic until the new CK2 update and dlc rolls out and then tweak it. Since it is part of the upcoming expansion "Holy Fury" not everyone will be able to use it and it actually is related to one character only so the "house" trait looks pretty different to me than the new bloodline system of PDX.

**Comment by blackninja9939** [comment]
Mods are free to use the bloodline system to make their own bloodlines.

It is also not attached to one character only, multiple people can be part of the bloodlines of course.

**Comment by TitanRogue** [comment]
@blackninja9939 
Of course multiple characters can be part of a bloodline. I meant to say that the PDX bloodline system is focusing on characters only and the trait on the house and that the bloodline feature is DLC exclusive.

**Comment by blackninja9939** [comment]
As I said in the same message mods are free to make their own bloodlines.

Your traits do not focus on the house either, they are character based and do not tie into the dynasty outside of the naming (which often can fail due to being too dynamic). Bloodlines are just a better way of doing all of that.

**Comment by RowanMaBoot** [comment]
black is right on the bloodline. The trait was set up the way it was to allow for any dynasty to use it without having to hard-define traits for every dynasty - but it has some issues.

It breaks if the great-great-grandfather doesn't exist. It also breaks when inherited too 'deep' into a dynastic branch (not inherently a bad thing, but this is part of the reason why males only inherit - it'd end up on virtually every character otherwise if left for enough time), as eventually the scoped character isn't say a Stark, but a Caverunner.

The alternative would be to hard-define house traits. But this means minor houses that become great will have to marry into a house just for a trait, and it will never be for their dynasty.

I was going to remove inheritance, and instead was going to use an event to award it, meaning men/women could get it, and it'd be under more control.

But then I saw the new bloodline feature being brought in by PDX. And given black is saying it is a moddable and can be used in mods (and thus I presume it is like off-map powers in that the feature can, in theory, be used by people without the DLC so long as they have a mod that has created bloodlines), then I see no reason to not use the new feature.
